win/CS/HandBrakeWPF/ViewModels/SubtitlesViewModel.cs

```diff
@@ -782,6 +808,44 @@ private void AddInputSubtitles(string[] filenames)
             }
         }
 
+        private static IEnumerable<string> GetExternalSubtitlesBasedOnFileName(string directoryName, string fileName)
+        {
+            var subtitleFiles = Directory.EnumerateFiles(directoryName, $"{fileName}.*.ass").ToList();
+            subtitleFiles.AddRange(Directory.EnumerateFiles(directoryName, $"{fileName}.*.srt").ToList());
+            subtitleFiles.AddRange(Directory.EnumerateFiles(directoryName, $"{fileName}.*.ssa").ToList());
+            return subtitleFiles;
+        }
+
+        private void AddExternalSubtitles(string directoryName, string fileName)
+        {
+            foreach (string subtitleFile in GetExternalSubtitlesBasedOnFileName(directoryName, fileName))
+            {
+                var languageCode = GetLanguageCodeFromExternalSubtitleFile(subtitleFile);
+
+                var languageNames = LanguageUtilities.GetLanguageNames(new List<string>() { languageCode });

```

Comment:
is there a defined standard in language codes in subtitle filenames?  We use ISO639 internally but there are other standards that'll fail lookups with this.